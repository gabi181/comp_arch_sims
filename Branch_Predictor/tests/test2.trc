16 7 24 2 global_history global_tables using_share_mid
0x71B28C08 T 0x2250B384
0x11BAA82C N 0x7AE6E62C
0x01794964 T 0x3EBB8644
0x71B28C08 N 0x2250B384
0x71B28C08 N 0x2250B384
0x11BAA82C N 0x7AE6E62C
0x23EF8D5C T 0x3F97EC5C
0x23EF8D5C N 0x3F97EC5C
0x71B28C08 N 0x2250B384
0x23EF8D5C N 0x3F97EC5C
0x11BAA82C T 0x7AE6E62C
0x71B28C08 T 0x2250B384
0x11BAA82C N 0x7AE6E62C
0x01794964 T 0x3EBB8644
0x01794964 N 0x3EBB8644
0x71B28C08 T 0x2250B384
