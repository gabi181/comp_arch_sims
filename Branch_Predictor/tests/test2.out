0x71b28c08 N 0x71b28c0c
0x11baa82c N 0x11baa830
0x1794964 N 0x1794968
0x71b28c08 T 0x2250b384
0x71b28c08 T 0x2250b384
0x11baa82c T 0x7ae6e62c
0x23ef8d5c N 0x23ef8d60
0x23ef8d5c T 0x3f97ec5c
0x71b28c08 T 0x2250b384
0x23ef8d5c T 0x3f97ec5c
0x11baa82c T 0x7ae6e62c
0x71b28c08 T 0x2250b384
0x11baa82c T 0x7ae6e62c
0x1794964 T 0x3ebb8644
0x1794964 T 0x3ebb8644
0x71b28c08 T 0x2250b384
flush_num: 11, br_num: 16, size: 1143b
